bin/move-and-rebuild-host-hil.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
```

Comment:
The code (my code) in general is ugly and bad (but working). Orran said I should stop working on this minimalistic version and join the others because that's the aim of the class.